Model/Capabilities.php

```diff
@@ -36,4 +36,12 @@ public function retrieveInstallmentBackends()
     {
         return $this->capabilitiesAPI->getInstallmentBackends();
     }
+
+    /**
+     * @return bool
+     */
+    public function isZeroInterests()
```

Comment:
and again

Reply:
Missed?

Reply:
that has been fixed some time ago @jonrandy I think you look at outdated stuff as even here.. is written
![image](https://user-images.githubusercontent.com/10651523/58938006-44b24280-8774-11e9-8f1a-d848c2113118.png)